src/pages/Profile/EditProfile/index.js

```diff
@@ -0,0 +1,82 @@
+import {editProfile} from "./index.tmpl";
+import "../index.css"
+import "../../..//components/button/index.tmpl"
+import "../../..//components/button/index.css"
+import "../../..//components/avatar/index.tmpl"
+import "../../..//components/avatar/index.css"
+import "../../..//components/input/index.tmpl"
+import "../../..//components/input/index.css"
```

Comment:
Как и с кавычками - так и с `;`. Если используешь в проекте, то старайся использовать везде. Чтобы код был написан в едином стиле
```suggestion
import {editProfile} from "./index.tmpl";
import "../index.css";
import "../../..//components/button/index.tmpl";
import "../../..//components/button/index.css";
import "../../..//components/avatar/index.tmpl";
import "../../..//components/avatar/index.css";
import "../../..//components/input/index.tmpl";
import "../../..//components/input/index.css";
```